scripts/generate-openapi.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -ux
+SCRIPT_DIR="$(
+  cd -- "$(dirname "$0")" >/dev/null 2>&1 || exit
+  pwd -P
+)"
+PROJECT_ROOT="${SCRIPT_DIR}"/..
+SWAGGER_SPEC_FILE="${PROJECT_ROOT}/transcendence-app/swagger-spec.yaml"
+mkdir -p "${PROJECT_ROOT}"/webapp/src/shared/generated
+ME="$(whoami)"
```

Comment:
what do you think about using $USER instead? you'll always have the current user assigned to the env

Reply:
Yes, but this $MY_USER would be the user that ran the docker run, passed as an env variable from the docker run command, not the user that is running the commands inside the docker container. The idea is: normally the user inside the container is root or sudoer. If it creates files/folders in a volume it shares with the parent system, and then quits execution, the generated files will be owned by root, and you won't be able to update or delete them without sudo, which is inconvenient. This is a well known docker problem, and this code segment is a workaround.